File: js/physicx/src/law.js
import Constants from "./constants.js";

const Law = new function () {
	this.Hubble = function (D) {
		return D * Constants.H_0;
	}
	this.Planck = function (nu, T) {
		return (8 * Math.PI * nu ** 2 * Constants.h * nu) / (Constants.c ** 3 * Math.exp((Constants.h * nu) / (Constants.k_B * T)) - 1);
	}
}

export default Law;

File: python/src/physicX/law.py
from .constants import *

## Physics law's 

### Hubble law's 
def hubble(D):
	return H_0 * D

### Planck law's
def planck(nu, T):
	return (8 * np.pi * nu**2 * h * nu)/(c**3 * np.exp((h * nu)/(k_B * T)) - 1)
